assets/styles/jon.scss

```diff
@@ -0,0 +1,8 @@
+#test{
+  .name1{
+    border: 1px solid #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.